src/queries/rum-pageviews.sql

```diff
@@ -39,7 +39,7 @@ BEGIN
       END AS date
     FROM helix_rum.PAGEVIEWS_V3(
       inurl, # url
-      (inoffset * ingranularity) - 1, # offset
+      GREATEST((inoffset * ingranularity) - 1, 0), # offset
```

Comment:
Since there's only two values to be compared, MAX would work just fine, GREATEST sounds more appropriate for a serie of args (2+).
```suggestion
      MAX((inoffset * ingranularity) - 1, 0), # offset
```

Reply:
MAX is an aggregate function in BigQuery, it will not accept two params.  You might be thinking of javascript Math.max().  Try `select max(1, 0)` and see.

https://cloud.google.com/bigquery/docs/reference/standard-sql/aggregate_functions#max

Reply:
I thought MAX was taking multiple parameters and that GREATEST and MAX could be swapped, thanks for clarifying!